changelog/bug-1597331.md

```diff
@@ -0,0 +1,4 @@
+level: patch
+reference: bug 1597331
+---
+Instances created by the AWS and Google providers are now tagged/labeled with the workerPoolId.
```

Comment:
they were tagged with workerPoolId _before_ tho.... what I really see in the change, for aws provider you make it _absolutely clear_ what is workerPoolId, and for google provider you add `owner` and `created by` tags, as well as changing the regex rules for converting those 🤔

#preferential

Reply:
Good point - I'll redraft the changelog and land.